packages/core/src/utils/useFloatingUI/useFloatingUI.tsx

```diff
@@ -40,11 +40,11 @@ export interface FloatingComponentProps
   /**
    * Position props for the floating component
    */
-  top: number;
-  left: number;
+  top?: number;
```

Comment:
Why was this change needed? What is the underlying issue here?

Reply:
the Drawer isn't positioned relative to a trigger element , so these props don't need to be passed down

Reply:
If that's the case, let's add some JSDoc on the props, to help others to decide to use them or not.

Reply:
added clarification `Position props for the floating component. `top`, `left`, and `position` are optional for components where their floating elements are not positioned relative to a trigger element, i.e. Drawer, Dialog`